hs_core/models.py

```diff
@@ -23,6 +23,8 @@
 from languages_iso import languages as iso_languages
 from dateutil import parser
 import json
+from hs_core import HSAlib
```

Comment:
Yes, will work on it ASAP after I addressed all your comments. Thanks @ironfroggy